_posts/2017-12-18-XPDays-Benelux-2017.md

```diff
@@ -0,0 +1,208 @@
+* **Leadership and Engagement:** Interprets people's interior experiences and focuses on "I"
+* **Culture and Shared Vision:** Interprets the collective consciousness of a group of people and focuses on "We"
+* **Behavior and Metrics:** Observation of the behavior of people and focuses on "It"
+* **Organizational Architecture and Systems:** Focuses on the behavior of a group of people as functional entities seen from outside: "They"
+
+The following image gives an overview of the questions that help organize knowledge into the quadrants:
+<div class="row">
+  <div class="6u -3u 8u(small) -2u(small) 12u(xsmall)">
+    <img class="image fit" alt="Integral Theory" src="/img/xpdays-benelux-2017/Integral-Theory2.png">
+  </div>
+</div>
+
+After explaining Integral Theory, we picked up our post-it notes and organized them into the quadrants that were put up across the room.
+When the post-it notes were assigned to their respective quadrant, Frederik and Johannes asked us to move towards the quadrant in the room that we felt we had the most experience in or we felt we were good at.
+Next, we had to discuss in our 'quadrant groups' if the post-it notes that were put up in the quadrant were a good match for it or if they belonged in another quadrant.
+The final exercise consisted of moving with your group to the next quadrants and listening to one person, that stayed behind from the initial group, explain the conclusions they reached during the initial discussion.
+
+The presentation of Frederik and Johannes can be found [here](https://www.slideshare.net/frederikvannieuwenhuyse/an-integral-view-on-agile){:target="_blank"}.
+
+### The Art of Hosting Conversations that Matter - Johan Decoster & Jef Cumps
+
+<span class="image left"><img class="p-image" alt="Johan Decoster" src="/img/xpdays-benelux-2017/Johan-Decoster.png"></span>
+<span class="image left"><img class="p-image" alt="Jef Cumps" src="/img/xpdays-benelux-2017/Jef-Cumps.png"></span>
+[Johan Decoster](https://twitter.com/johan_decoster){:target="_blank"} is an Agile coach and trainer trying to make a difference in the lives of the people he works with by uncovering everyone’s unique potential and looking deeper into the essence behind any theory or concept.<br><br>
+[Jef Cumps](https://twitter.com/jcumps){:target="_blank"} is a very experienced coach and trainer supporting organisations in their transformation towards more agility and towards a more engaging, humane and effective way of looking at work.
+
+The session by Johan and Jef started with a small exercise. We paired up with our neighbor and asked each other the question, 'Why are you really at this conference, and what has it meant to you so far?'
+The point of the interview was asking questions to guide the interviewee in telling their story. After these interviews, the person interviewing had to share what they had learned with the rest of the table and the others had to take notes of the summary.
```

Comment:
what he or she had learned with the rest of us.
We were then asked to take notes of the summary.